ui/src/components/questions/post.js

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Markdown from '../helpers/markdown';
+import moment from 'moment';
+import Certificate from '../utils/certificate';
+import * as User from '../../models/user';
+import Button from '../forms/button';
+import { UserContext } from '../../usercontext';
+import Trophy from '../utils/trophy';
+import { t } from 'ttag';
+import Dialog from '../utils/dialog';
+import InputField from '../forms/inputfield';
+
+class Post extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            votes: 0,
+            isDeleteDialogOpen: false,
+            isEditDialogOpen: false,
+            currentBody: '',
+            postingToServer: false
+        };
+    }
+
+    componentDidMount() {
+        this.setState({
+            votes: this.props.votes
+        });
+    }
+
+    getClasses() {
+        let classes = `question-post ${this.props.className}`;
+        if (this.state.votes < -3) {
+            classes = `${classes} faded`;
+        }
+        if (this.props.answered) {
+            classes = `${classes} accepted`;
+        }
+        return classes;
+    }
+
+    incrementVotes() {
+        this.setState({
+            votes: this.state.votes + 1
+        });
+    }
+
+    decrementVotes() {
+        this.setState({
+            votes: this.state.votes - 1
+        });
+    }
+
+    getTime() {
+        return moment(this.props.created).fromNow();
+    }
+
+    renderAnswered() {
+        return (
+            <Certificate active={this.props.answered} />
```

Comment:
maybe a more precise name?

Reply:
It's a certificate symbol, so.. it's pretty accurate I would say. What did you have in mind?